### DISCUSSION TITLE
Application

### DISCUSSION THREAD

**Discussion by IsaacHunt** [nuxsmin/sysPass#1764] [Q&A] [answered]
Hey, 

I have looked around at documentations etc and I can seem to find an answer, wondering if someone know this answer.... 

Are the devs looking too add in an application for Windows/MAC in the future or is it purely via browser?

Sorry if stupid, new too this. On a big learning curve currently

**Comment by bitking** [comment, marked as answer]
No, its totally not viable to use a native windows/mac application when the scope of the project is a webapplication.

Please read and understand into having a LAMP server and the prupose of it and understand the fact that this project is run by 1 developer doing this in his free time.

**Reply by nuxsmin** [reply]
Right this is a web application and there isn't any development to make it PWA.
